containers/kubernetes/reference-content/introduction-to-kubernetes.mdx

```diff
@@ -143,25 +144,22 @@ Another type of pod controller is called DaemonSet. It ensures that all (or some
 
 ### Jobs and CronJobs
 
-Jobs manage a task until it runs to completion.  
+Jobs manage a task until it runs to completion. They can run multiple Pods in parallel, and are useful for batch-orientated tasks.
 
-Jobs are able to run multiple Pods in parallel, and they are useful for batch-orientated tasks.
-
-CronJobs in Kubernetes work like traditional cron jobs in Linux. They can be used to run tasks at a specific time or interval and may be useful for Jobs such as backups or cleanup tasks.
+CronJobs in Kubernetes work like traditional cron jobs on Linux. They can be used to run tasks at a specific time or interval and may be useful for Jobs such as backups or cleanup tasks.
```

Comment:
Is it right that Jobs at the end of the sentence is captialized?

Reply:
```suggestion
Cronjobs in Kubernetes work like traditional cron jobs on Linux. They can be used to run tasks at a specific time or interval and may be useful for Jobs such as backups or cleanup tasks.
```